Docs(GraphQL): Emphasize that @search annotation is required for filtering on predicates (#247)
dgraph-io/dgraph-docs: content/graphql/queries/search-filtering.md

}
```

### Query that filters objects by predicate

> Before filtering an object by a predicate, you need to add a @search directive to the field that will be used to filter the results.

Reviewer: ```suggestion
Before filtering an object by a predicate, you need to add a `@search` directive to the field that will be used to filter the results.
```

> For example, if you wanted to query events between two dates, or events that fall within a certain radius of a point, you could have an Event schema like this:

Reviewer: ```suggestion
For example, if you wanted to query events between two dates, or events that fall within a certain radius of a point, you could have an `Event` schema, as follows:
```

```
type Event {
  id: ID!
  date: DateTime! @search
  location: Point @search
}
```

> The search directive would allow filtering events that fall between dates like this:

Reviewer: ```suggestion
The search directive would let you filter events that fall within a date range, as follows:
```

```
query {
  queryEvent (filter: { date: { between: { min: "2020-01-01", max: "2020-02-01" } } }) {
    id
  }
}
```

> And events that have a location near a certain point can be filtered like this:

Reviewer: ```suggestion
You can also filter events that have a location near a certain point with the following query:
```
